experimental/dds/tree2/src/feature-libraries/chunked-forest/fence.json

```diff
@@ -1,5 +1,4 @@
 {
-	"tags": ["chunked-forest"],
```

Comment:
I'm not sure what good fences does with no tags specified, but we should probably leave this alone.

Reply:
added tag back in